janus_core/helpers/janus_types.py

```diff
@@ -121,6 +121,7 @@ class CorrelationKwargs(TypedDict, total=True):
 Ensembles = Literal["nph", "npt", "nve", "nvt", "nvt-nh", "nvt-csvr"]
 Properties = Literal["energy", "stress", "forces", "hessian"]
 PhononCalcs = Literal["bands", "dos", "pdos", "thermal"]
+Interpolators = Literal["ase", "pymatgen"]
```

Comment:
is not a third category, user provided?

Reply:
Yes, I did have something like `"custom"` or `None` originally as its own option, but since we can assume user provided/no interpolation if `band_structs` is passed (and it will raise errors if the initial/final structure are also passed), it felt a bit unnecessary to also make a user specify this too?

Reply:
what I am going at is that if we use user provided data, what will be the state of Interpolators? if None I am happy, if one of the two, I am not.

Reply:
Ah ok it was slightly inconsistent. The CLI set interpolator to be None, but actually the Python still required the user to choose None.

It now should set interpolator to None on the init for the Python or CLI, and docstrings should reflect this.